.github/settings.yml

```diff
@@ -26,4 +26,15 @@ collaborators:
   
   - username: mkorbi
     permission: admin
+    
+  - username: catblade
```

Comment:
Suggestion: put in alphabetical order:
- catblade
- cdeliaRH
- mkorbi
- nikimanoledaki
- scottrigby